golang/cosmos/app/upgrade.go

```diff
@@ -14,11 +14,8 @@ import (
 )
 
 var upgradeNamesOfThisVersion = []string{
-	"UNRELEASED_BASIC", // no-frills
-	"UNRELEASED_A3P_INTEGRATION",
-	"UNRELEASED_main",
-	"UNRELEASED_devnet",
-	"UNRELEASED_REAPPLY",
+	"agoric-upgrade-18",
+	"agorictest-upgrade-18",
```

Comment:
Are we not releasing variants for price feed purposes?

Reply:
as per my understanding, both `replace-electorate-core.js` and `updatePriceFeeds.js` used the same variants. But now that I have double-checked, `updatePriceFeeds` isn't using `MAINNET` and `DEVNET` as keys in its configuration.

```
const configurations = {
  A3P_INTEGRATION: {
    oracleAddresses: [
      'agoric1ee9hr0jyrxhy999y755mp862ljgycmwyp4pl7q', // GOV1
      'agoric1wrfh296eu2z34p6pah7q04jjuyj3mxu9v98277', // GOV2
      'agoric1ydzxwh6f893jvpaslmaz6l8j2ulup9a7x8qvvq', // GOV3
    ],
    inBrandNames: ['ATOM', 'stATOM'],
  },
  main: {
    oracleAddresses: [
      'agoric144rrhh4m09mh7aaffhm6xy223ym76gve2x7y78', // DSRV
      'agoric19d6gnr9fyp6hev4tlrg87zjrzsd5gzr5qlfq2p', // Stakin
      'agoric19uscwxdac6cf6z7d5e26e0jm0lgwstc47cpll8', // 01node
      'agoric1krunjcqfrf7la48zrvdfeeqtls5r00ep68mzkr', // Simply Staking
      'agoric1n4fcxsnkxe4gj6e24naec99hzmc4pjfdccy5nj', // P2P
    ],
    inBrandNames: ['ATOM', 'stATOM', 'stOSMO', 'stTIA', 'stkATOM'],
    contractTerms: { minSubmissionCount: 3 },
  },
  devnet: {
    oracleAddresses: [
      'agoric1lw4e4aas9q84tq0q92j85rwjjjapf8dmnllnft', // DSRV
      'agoric1zj6vrrrjq4gsyr9lw7dplv4vyejg3p8j2urm82', // Stakin
      'agoric1ra0g6crtsy6r3qnpu7ruvm7qd4wjnznyzg5nu4', // 01node
      'agoric1qj07c7vfk3knqdral0sej7fa6eavkdn8vd8etf', // Simply Staking
      'agoric10vjkvkmpp9e356xeh6qqlhrny2htyzp8hf88fk', // P2P
    ],
    inBrandNames: ['ATOM', 'stTIA', 'stkATOM'],
  },
};
```
so the variant name from `getVariantFromUpgradeName` wouldn't work for price feeds on devnet and mainnet. What do you think?

Reply:
@Chris-Hibbert can you clarify?

Reply:
I've created #10363 to repair this.